spool_1/boolean_spec.rb

```diff
@@ -18,34 +18,34 @@
 
   describe "falsiness" do
     it "is false for false" do
-      expect(__).to be_false
+      expect(__).to be_falsey
```

Comment:
What I mean is, the test description says `false for false` when it is actually `falsey for false`.